pset-functions.js

```diff
@@ -115,6 +188,26 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function isPalindrome(string){
+
+  let reverseString = string.split('').reverse('').join('')
+
+  if (reverseString === string) {
+    return true
+  } else {
+    return false
+  }
```

Comment:
Again great example of method chaining on line 193. You can take advantage of a conditional statement returning a true/false value (as shown [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L174)).